Error installing on Node v13.1.0

Apparently there are some issues with NodeJs newer version.

When installing `node-sass` dependency, a lot of compiling errors are happening:

```sh
../../nan/nan_converters_43_inl.h:18:51: error: no matching function for call to ‘v8::Value::ToBoolean(v8::Local<v8::Context>)’
```
...
```sh
../../nan/nan_converters_43_inl.h:37:57: error: no matching function for call to ‘v8::Value::BooleanValue(v8::Local<v8::Context>)’
   return val->NAME ## Value(isolate->GetCurrentContext()); 
```

Tested with version 10.x and everything is OK.

--- a/package.json
+++ b/package.json
@@ -154,7 +154,7 @@
     "jest": "^23.6.0",
     "jest-serializer-vue": "^2.0.2",
     "node-notifier": "^5.3.0",
-    "node-sass": "^4.10.0",
+    "node-sass": "^4.13.0",
     "npm-run-all": "^4.1.5",
     "onchange": "^5.1.3",
     "optimize-css-assets-webpack-plugin": "^5.0.1",

